support/latest/policies/containerization.md

```diff
@@ -1,21 +1,19 @@
 
-The above illustration shows four deployment options. Alfresco creates and supports only the software represented as App1, App2, and App3. Alfresco doesn’t create or support anything else in the diagram - that’s considered to be “infrastructure”.
+From time to time Alfresco will create and make available reference deployment scripts to help people be successful with a couple of the more-popular deployment tools. These reference scripts are supported by Alfresco, and if a defect is found in them or an enhancement is wanted then a ticket can be raised through Alfresco Support. The customer is expected to know how to use the deployment technology associated with these scripts, and they shouldn’t use them if they don’t. Alfresco is unable to offer training in the use of deployment tools.
 
-There are different ways to deploy Alfresco software into an infrastructure before it can run. A few options include using custom shell scripts, installers, Ansible, Terraform, Docker Compose, Kubernetes, Docker Enterprise, OpenShift, Heroku, Chef, Puppet, Rancher, AWS CloudFormation, Azure Resource Manager, Google Cloud Deployment Manager, AWS ECS, AWS EKS, and so on.
+Because everyone’s infrastructure is different to everyone else’s - and everyone has their own set of deployment guidelines and best practices - when Alfresco does offer deployment scripts they’re only references. Each customer is expected to adjust them to suit its own purpose. Alfresco makes no representation that these reference scripts will meet any particular customer's specific standards.
 
-Alfresco doesn’t create or support any deployment tool and customers may choose the deployment tool they feel most comfortable with, often this will be based on a company standard. Customers must have the skills and knowledge needed to use their chosen deployment tool effectively. Alfresco cannot provide assistance on the deployment tool/mechanism, our approach is to support industry-leading standards and best practices to give customers the choice of platforms on which to deploy.
+When Alfresco provides reference images (either virtual machine images or container images) it will aim to update those images as the underlying operating system and libraries are released. However, as those images are built upon the work of others, Alfresco cannot give guarantees and assurances on the timeliness of upgrades and patches for security issues in those underlying components. As with any reference deployment scripts, any virtual machine or container images provided by Alfresco are considered references from which customers may derive their own, incorporating their own specific guidelines and best practices as needed.
```

Comment:
"images provided by Alfresco are considered references from which customers may derive their own" - This implies that we have some sort of SDK or documentation that tells them how they can use ours to derive their own. I'm not sure that is the case at the moment.

Reply:
We should provide the source used to create the images (the Dockerfiles, for example). Although we do have them in GitHub, it isn't the simplest thing to locate them. Making this easier would be useful. We shouldn't aim to educate people in how to create images.